packages/client/src/components/TransactionStatusIcon.js

```diff
@@ -3,17 +3,17 @@ import React from "react";
 const STATUS_TO_BACKGROUND_COLOR = {
   confirmed: "#18AA01",
   pending: "#FF8A00",
+  rejected: "#CD0B0B",
```

Comment:
@KsenijaZivkovic added a status Icon and colors in her pr, https://github.com/DapperCollectives/VESSEL/pull/108
I think this file can be deprecated now.

Reply:
would rather update this in a follow up PR after #108 is merged to not be dependent on it, or if it gets merged before this PR does I can update